sagemaker-python-sdk/pytorch_batch_inference/sagemaker_batch_inference_torchserve.ipynb

```diff
@@ -0,0 +1,463 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    image_uri = "763104351884.dkr.ecr.us-west-2.amazonaws.com/pytorch-inference:1.9.0-gpu-py38-cu111-ubuntu20.04"
</pre><p>Don't hard-code the image_uri with region. Use <code>sagemaker.image_uris.retrieve()</code>  method.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/aws/amazon-sagemaker-examples/pull/3008/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
<p>Done</p>

Reply:
<p>Programmatically determine region instead of hard-coding us-west-2, such as:</p><pre class="ql-syntax" spellcheck="false">session = boto3.session.Session()
region = my_session.region_name
</pre>

Reply:
Done, thanks